lib/services.js

```diff
@@ -2,7 +2,12 @@ var fs = require('fs');
 var ejs = require('ejs');
 
 ejs.filters.q = function(obj) {
-  return JSON.stringify(obj, null, 2 );
+  // to handle strings being returned with double-quotes
+  // but linter requires single quotes for strings
+  if (typeof obj == 'string' && !/\\'/.test(obj))
+    return '\'' + obj + '\'';
+  else
+    return JSON.stringify(obj, null, 2);
```

Comment:
do not change the code logic, the only difference should be the space at the end of the 2

```
return JSON.stringify(obj, null, 2);
```

The other stuff you're changing is untested and should be done in a separate PR.